zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1965,6 +1968,8 @@ public <T> T retryUntilConnected(final Callable<T> callable)
     if (_monitor != null) {
       _monitor.increaseOutstandingRequestGauge();
     }
+    int retryCount = 1;
```

Comment:
nit: not a big deal, but should we initialize it with 0? And change below `retryCount++` to `++retryCount`